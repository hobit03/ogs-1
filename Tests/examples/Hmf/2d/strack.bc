GINA - Boundary Condition
#BOUNDARY_CONDITION
 $PCS_TYPE
  LIQUID_FLOW
 $PRIMARY_VARIABLE
  PRESSURE1
 $GEO_TYPE
  POLYLINE PLY_LEFT
 $DIS_TYPE
  CONSTANT   496465.0
#BOUNDARY_CONDITION
 $PCS_TYPE
  LIQUID_FLOW
 $PRIMARY_VARIABLE
  PRESSURE1
 $GEO_TYPE
  POLYLINE PLY_RIGHT
 $DIS_TYPE
  CONSTANT   -496465.0
#STOP
