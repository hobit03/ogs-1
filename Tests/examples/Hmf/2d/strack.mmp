#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  2
 $GEOMETRY_AREA
  1.0
 $POROSITY
  1  0.2
 $STORAGE
  1  0.0 
 $TORTUOSITY
  1  1.000000e+000
 $PERMEABILITY_TENSOR
  ISOTROPIC   1.0e-12
 $HEAT_DISPERSION
  1  0.000000e+000 0.000000e+000
#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  1
 $GEOMETRY_AREA
  0.05
;med  0.04330127
;mean: 0.03815201
;max:  0.05
 $POROSITY
  1  0
 $STORAGE
  1  0.0 
 $TORTUOSITY
  1  1.000000e+000
 $PERMEABILITY_TENSOR
  ISOTROPIC   1.0e-10
 $HEAT_DISPERSION
  1  0.000000e+000 0.000000e+000
#STOP
