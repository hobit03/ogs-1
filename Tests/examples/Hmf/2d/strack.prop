<?xml version="1.0"?>
<coupling>
	<P algorithm="Serial" convergence="FemFunctionConvergenceCheck" max_itr="1" epsilon="1e-4">
		<out>PRESSURE</out>
		<out>VELOCITY</out>
		<problems>
			<M name="LIQUID_FLOW">
				<out>PRESSURE1</out>
			</M>
			<M name="PRESSURE_TO_ELEMENT_VELOCITY">
				<in>PRESSURE</in>
				<out>VELOCITY</out>
			</M>
		</problems>
	</P>
</coupling>

