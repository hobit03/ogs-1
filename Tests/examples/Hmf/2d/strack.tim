Gina - TIM-Datei
#TIME_STEPPING
  $PCS_TYPE
    LIQUID_FLOW
  $TIME_STEPS
    1 1
  $TIME_END
    1
  $TIME_START
    0
#STOP
