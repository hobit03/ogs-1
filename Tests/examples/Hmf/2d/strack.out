Gina - Out-Datei
#OUTPUT
 $NOD_VALUES
   PRESSURE1
   VELOCITY_X1
   VELOCITY_Y1
   VELOCITY_Z1
 $ELE_VALUES
   VELOCITY1_X
 $GEO_TYPE
   DOMAIN 
 $DAT_TYPE
   PVD
 $TIM_TYPE
   STEPS 1
#STOP
